pumpkin-inventory/Cargo.toml

```diff
@@ -7,6 +7,9 @@ edition.workspace = true
 # For items
 pumpkin-world = { path = "../pumpkin-world" }
 
+simple_logger = { version = "5.0.0", features = ["threads"] }
```

Comment:
Thats definitely not needed here

Reply:
you have a dbg! statement there so I changed it to log::dbg!
unless you're referring to the threads feature, that's probably not needed